Refresh.GameServer/Database/GameDatabaseContext.Contests.cs

```diff
@@ -52,6 +52,15 @@ public IEnumerable<GameContest> GetAllContests()
             .FirstOrDefault();
     }
     
+    public GameContest? GetLastActiveContest()
+    {
+        DateTimeOffset now = this._time.Now;
+        return this.GameContests
+            .Where(c => c.EndDate <= now)
+            .OrderByDescending(c => c.EndDate)
+            .FirstOrDefault();
+    }
```

Comment:
When i read the name "get last active contest" I imagine that it would get the contest which was most recently active (including currently running contests), i think `GetLatestCompletedContest` is a much more clear name